test/sys/test_statfs.rs

```diff
@@ -93,7 +92,6 @@ fn assert_fs_equals_strict(fs: Statfs, vfs: Statvfs) {
     assert_eq!(fs.files_free() as u64, vfs.files_free() as u64);
     assert_eq!(fs.blocks_free() as u64, vfs.blocks_free() as u64);
     assert_eq!(fs.blocks_available() as u64, vfs.blocks_available() as u64);
-    assert_eq!(fs.files() as u64, vfs.files() as u64);
```

Comment:
You should leave the check in `assert_fs_equals_strict`.  This function contains other assertions that are even more likely to fail, but it's only called from `#[ignore]` tests.

Reply:
done